Integrations/SplunkPy/SplunkPy.yml

```diff
@@ -259,8 +281,61 @@ script:
     - contextPath: Splunk.Raw.Parsed
       description: The raw event data (parsed).
       type: unknown
+  - arguments:
+    - default: false
+      description: |-
+        Event payload key-value. Value can be a string or a JSON object.
+        JSON example: {"event": {"message":"Access log test message"}}
+        String example: "event": "Access log test message."
+      isArray: false
+      name: event
+      required: true
+      secret: false
+    - default: false
+      description: Fields for indexing that do not occur in the event payload itself.
+      isArray: false
+      name: fields
```

Comment:
Is it CSV list of fields?

Reply:
No. any field can come here. 
It will add it to the event as the following:
![image](https://user-images.githubusercontent.com/45915502/72682618-ad69e780-3ad7-11ea-8e45-d459857c5171.png)

Reply:
How do the user expected to pass the list of fields.

Reply:
Comma separated. will update the description and add an example to the docs afterwards